### DISCUSSION TITLE
Recently played tab resetting after each map.

### DISCUSSION THREAD

**Discussion by simmdoge** [ppy/osu#28783] [Ideas] [answered]
Ok so when playing osu!, i tend to filter beatmaps via the recently played tab, the main problem i have with doing this is that after each map it resets my filter to the "today" option, as in the maps ive played today.

Is there no way there could be some sort of option added in order to keep my filter to stay on the last map that i played? 
so for example, if i play something from last month, could you not just have the game automatically go onwards to the beatmap ahead of what i just played instead of putting all the way back to the top again?

this probably doesnt effect many people, but its a major annoyance for me when playing the game.

**Comment by Joehuu** [comment, marked as answer]
This sounds like a stable issue, which is feature-locked.
